_topics/en/adaptor-signatures.md

```diff
@@ -18,153 +20,164 @@ categories:
 ## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
 ## Should be less than 500 characters
 excerpt: >
-  **Adaptor signatures** are auxiliary signature data that commit to a
-  hidden value.  When an adaptor signature is combined with a final
-  signature, it reveals the hidden value.  Alternatively, when combined
-  with the hidden value, it reveals the final signature.  The commitment
-  may be reused in other signatures without the secondary signers
-  knowing the hidden value or their own final signature.
+  **Adaptor signatures** (also called **signature adaptors**) are
```

Comment:
Did you mean to write that the other way round I think?

Reply:
For now, I've left the page title as "adaptor signatures", so I think it makes sense for that to be the primary term in the description.  If we decide to retitle the page, I'll reverse the term and the alias.

Reply:
Oh sorry I didn't read carefully what you wrote in the comment above. OK got it, thanks. And yes :+1: from me for 'signature adaptors', although the confusion will continue into perpetuity is my prediction :)